CiarnuroRT/webapp/style.css

```diff
@@ -300,23 +300,23 @@ div.list > div.entry > div.rightButtons > a.button.remove{
 	background-position:50% 50%;	
 }
 div.list > div.entry > div.content > div.small{
-    font-size:0.85em;
```

Comment:
I'm not sure this change is correct, this would set it to the default size.

Reply:
yes, because i change a bit the dimension but now I have restored the old dimension.